server/journeys/referral/view.njk

```diff
@@ -0,0 +1,6 @@
+{% block content %}
+
+  <h1>Referrals is under construction...</h1>
+  <p>Please check back later when there is content to view.</p>
+
+{% endblock %}
```

Comment:
better for this to extends from layout.njk?

anyway this is just a place holder at the moment